q2_diversity/_beta/_method.py

```diff
@@ -110,19 +113,34 @@ def beta_phylogenetic_alt(table: BIOMV210Format, phylogeny: NewickFormat,
              variance_adjusted=variance_adjusted, bypass_tips=bypass_tips)
 
 
-def beta(table: biom.Table, metric: str, n_jobs: int=1)-> skbio.DistanceMatrix:
-    if metric not in non_phylogenetic_metrics():
+def aitchison(x, y, **kwds):
+    return euclidean(clr(x), clr(y))
+
+
+def beta(table: biom.Table, metric: str,
+         pseudocount: int=1, n_jobs: int=1)-> skbio.DistanceMatrix:
+
+    if not (metric in non_phylogenetic_metrics()):
         raise ValueError("Unknown metric: %s" % metric)
+
+    if metric == 'aitchison':
```

Comment:
Style comment/ How about something like this:

```python
counts = table.matrix_data.toarray().astype(int).T
if metric == 'aitchison':
    counts = counts + psuedocount
    metric = aitchison

That code is untested, but it might be easier for new readers of the codebase.

Reply:
As well, if we are introducing new pseudocount functionality, a test or two should probably be added to cover this case.